### PR TITLE
Weston/volunteer page

### PR DESCRIPTION
**Pull request by wheninseattle**
## What

Create the new volunteer page for desktop and tablet viewing

## Why Do

Introduce additional content to the volunteer page and include new design standards.

## Show Me

20230708-Volunteer-New

### CODE REVIEW

File: pages/volunteers_new.tsx [hunk 1]
```diff
@@ -1,16 +1,261 @@
...
+          <Grid container spacing={2}>
+            {gridContent.map((item) => (
+              <Grid item xs={12} sm={6} key={item.label}>
+                <ListItemWithIcon listIcon={item.icon} listText={item.label} />
+              </Grid>
+            ))}
+          </Grid>
+          <Typography variant="bodyLarge">
+            All of our volunteers are vetted for experience, and sign a
+            volunteer agreement before commencing work with Open Seattle.
```
Comment (santina) [inline pages/volunteers_new.tsx:128]: oops I haven't signed it ^_^  can't blame me if I mess up 👿

File: pages/volunteers_new.tsx [hunk 2]
```diff
@@ -1,16 +1,261 @@
...
+        </Box>
+      </Container>
+      {/* Volunteer Roles Section */}
+      <Container maxWidth="md">
+        <Stack gap={8} sx={{ textAlign: 'center', paddingY: 8 }}>
+          <Typography variant="headlineLarge">
+            How you can participate
+          </Typography>
+          <Grid container spacing={2}>
+            {gridContent.map((item) => (
```
Comment (santina) [inline pages/volunteers_new.tsx:120]: nice!

File: pages/volunteers_new.tsx [hunk 3]
```diff
@@ -1,16 +1,261 @@
...
+                >{`${index + 1}.`}</Typography>
+                <Typography variant="bodyLarge" mx={2}>
+                  {item}
+                </Typography>
+              </li>
+            ))}
+          </ol>
+        </Stack>
+      </Container>
+      {/* Volunteer Application */}
```
Comment (santina) [inline pages/volunteers_new.tsx:236]: You can also break sections into components (optionally)

**Review by santina — APPROVED** [review]
Looks amazing! so much content. 
I couldn't find the volunteer agreement doc I wrote. if we find it we can add them to this page then.

File: pages/volunteers_new.tsx (hunk 3 above)
Comment (wheninseattle) [reply pages/volunteers_new.tsx:236]: Oh, that is a good idea. I'll try to implement that when I take care of the mobile version!

**Pull request merged**